src/nvme/util.c

```diff
@@ -402,7 +402,7 @@ void nvme_init_dsm_range(struct nvme_dsm_range *dsm, __u32 *ctx_attrs,
 }
 
 void nvme_init_id_ns(struct nvme_id_ns *ns, __u64 nsze, __u64 ncap, __u8 flbas,
-		__u8 dps, __u8 nmic, __u32 anagrpid, __u16 nvmsetid)
+		__u8 dps, __u8 nmic, __u32 anagrpid, __u16 nvmsetid, __u64 lbstm)
```

Comment:
As this is breaking the API, do we really need to add lbstm?  Actually I wonder if we shouldn't just remove the
function as nvme_id_ns has many members and this function is likely always missing out for someone. The only user for this function I found is in create_ns() in nvme-cli. We could easily ditch this function IMO.

Reply:
Taking it out and handling this in nvme-cli makes sense to me, removed the function.

Reply:
This should be a new commit. It has nothing to do with introducing the lbstm into struct nvme_id_ns. 
Also can nvme_init_id_ns also remove from the src/libnvme.map export file? Obviously nvme-cli needs a fix as well.

Thanks!